src/arch/pami-linux-ppc64le/charmrun

```diff
@@ -242,11 +242,11 @@ then
 		ncpus="#PBS -l ncpus=$nodes"
 		;;
 	esac
-	if test $USE_LSF -eq 0
+	if [[ "$USE_LSF" = '0' ]]
```

Comment:
Why not keep the `-eq` here?

Reply:
Partly for consistency. Also, my thinking is that a string comparison is less work than parsing the variable to an integer before comparing.